src/target/riscv/riscv-013.c

```diff
@@ -145,6 +145,62 @@ typedef struct {
 	struct target *target;
 } target_list_t;
 
+struct ac_cache {
+	uint32_t *commands;
+	size_t size;
+};
+
+static int ac_cache_elem_comparator(const void *plhs, const void *prhs)
+{
+	const uint32_t lhs = *(const uint32_t *)plhs;
+	const uint32_t rhs = *(const uint32_t *)prhs;
+	return (lhs > rhs) - (lhs < rhs);
```

Comment:
Really a minor detail: p_lhs and p_rhs is bit more readable than plhs, prhs.

```suggestion
static int ac_cache_elem_comparator(const void *p_lhs, const void *p_rhs)
{
	const uint32_t lhs = *(const uint32_t *)p_lhs;
	const uint32_t rhs = *(const uint32_t *)p_rhs;
	return (lhs > rhs) - (lhs < rhs);
```

Reply:
Addressed